subsys/net/lib/aws_fota/src/aws_fota_json.c

```diff
@@ -123,19 +110,13 @@ struct update_rsp_obj {
 static const struct json_obj_descr update_job_exec_stat_rsp_descr[] = {
 	JSON_OBJ_DESCR_PRIM_NAMED(struct update_rsp_obj, "status",
 			status, JSON_TOK_STRING),
```

Comment:
You might want to align this line in the same way as those below.

Reply:
done